crates/holo_hash/Cargo.toml

```diff
@@ -16,37 +16,46 @@ normal = ["tracing"]
 [dependencies]
 thiserror = "1.0.22"
 
-arbitrary = {version = "1.0", optional = true}
-base64 = {version = "0.13", optional = true}
-blake2b_simd = {version = "0.5.10", optional = true}
+arbitrary = { version = "1.0", optional = true }
+base64 = { version = "0.13", optional = true }
+blake2b_simd = { version = "0.5.10", optional = true }
 derive_more = { version = "0.99", optional = true }
 fixt = { version = "^0.2.0-beta-rc.0", path = "../fixt", optional = true }
-futures = {version = "0.3", optional = true}
-holochain_serialized_bytes = {version = "=0.0.51", optional = true }
+futures = { version = "0.3", optional = true }
+holochain_serialized_bytes = { version = "=0.0.51", optional = true }
 kitsune_p2p_dht_arc = { version = "^0.2.0-beta-rc.1", path = "../kitsune_p2p/dht_arc" }
-must_future = {version = "0.1", optional = true}
-rand = {version = "0.8.5", optional = true}
+must_future = { version = "0.1", optional = true }
+rand = { version = "0.8.5", optional = true }
 rusqlite = { version = "0.28", optional = true }
 serde = { version = "1", optional = true }
 serde_bytes = { version = "0.11", optional = true }
-tracing = { version = "0.1", optional = true}
+tracing = { version = "0.1", optional = true }
 
 [dev-dependencies]
-serde_json = { version = "1.0.51", features = [ "preserve_order" ] }
+serde_json = { version = "1.0.51", features = ["preserve_order"] }
 
 [features]
-
 default = ["serialization"]
 full = [
     "fixturators",
     "hashing",
     "encoding",
-    "rusqlite",
-    "kitsune_p2p_dht_arc/sqlite"
+    "no-deps"
```

Comment:
Most of the other crates still support linking against a system sqlite if you are careful. The configuration here does not seem to allow that.

I don't know if we care or not...

Reply:
I think there are two things to consider with that
1. The risk of a branched tree where some libs are getting linked to a system sqlite and others get linked to a bundled one which could lead to undefined behaviour. Probably quite a small risk though really
2. If we rely on the features that are enabled in the bundled sqlite then linking to a sqlite which may not have the same features enabled could break assumptions we are making. This actually seems to be the case with the sqlite that is packaged by default for Nix https://github.com/holochain/scaffolding/actions/runs/4733081304/jobs/8400083888. Yes we could configure that sqlite but we can't avoid people building outside of Nix and causing themselves trouble that will be very hard to track down

I think overall it's best to always bundle so you either opt into bundling or bundling+encryption and hopefully we get more reproducible builds that way